tests/comic_test.py

```diff
@@ -43,5 +43,10 @@ def test_invalid_isbn(self):
         murpg = self.m.comics({'id': 1143}).comics[0]
         self.assertEqual(murpg.isbn, '785110283')
 
+    def test_invalid_diamond_code(self):
+        """Sometimes Marvel API sends number for isbn"""
+        hulk = self.m.comics({'id': 27399}).comics[0]
+        self.assertEqual(hulk.diamond_code, '0')
```

Comment:
Are we sure int 0 shouldn't be cast to `None`? Is it really invalid or are they trying to signal something?

Reply:
Actually asking, I'm sure you know more about this quirk than I do.

Reply:
I think they are using manuel input or use something like an excell sheet in their process and that leads to typo in fields (fields are automatically type to numbers if there is only digits it seems). I'm not sure it would be a good idea to try to correct the data when there seems to be unusual case.